### PR TITLE
Use proper pathSeparator for the operating system

### PR DESCRIPTION
**Pull request by n2ux**
When running on Windows use the backslash as the path separator, other wise use the forward slash

### CODE REVIEW

File: command/token/helper_internal.go [hunk 1]
```diff
@@ -21,11 +21,16 @@ type InternalTokenHelper struct {
...
+		pathSeparator := "\\"
+	} else {
+		pathSeparator := "/"
 	}
-	i.tokenPath = homePath + "/.vault-token"
+        homePath, err := homedir.Dir()
+        if err != nil {
+                panic(fmt.Sprintf("error getting user's home directory: %v", err))
+        }
+        i.tokenPath = homePath + pathSeparator + ".vault-token"
```
Comment (kalafut) [inline command/token/helper_internal.go]: I _think_ `filepath.Join()` would take care of this in a cross platform way?

Comment (jefferai) [reply command/token/helper_internal.go]: Yep, good call.

**Comment by jefferai** [comment]
Please fmt the code properly -- it's got spaces there instead of tabs.

**Comment by jefferai** [comment]
Thanks!

**Pull request merged**